java/HealthVault Mobile/HVSample/src/main/res/drawable-hdpi/circle_background.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="oval">
+    <solid android:color="#86bbbf"/>
+    <stroke android:width="2dp" android:color="#ffffff" />
+    <size android:width="180dp" android:height="180dp"/>
```

Comment:
I have created task # 54619 for this comment

---
In reply to: [112366301](https://github.com/Microsoft/healthvault-samples/pull/4#discussion_r112366301) [](ancestors = 112366301)